packages/teams-js/src/internal/communication.ts

```diff
@@ -373,14 +378,18 @@ const sendRequestToTargetWindowHelperLogger = communicationLogger.extend('sendRe
  */
 function sendRequestToTargetWindowHelper(
   targetWindow: Window,
-  request: MessageRequestWithRequiredProperties | NestedAppAuthRequest,
+  messageRequest: MessageRequestWithRequiredProperties | NestedAppAuthRequest,
 ): MessageRequestWithRequiredProperties | NestedAppAuthRequest {
   const logger = sendRequestToTargetWindowHelperLogger;
   const targetWindowName = getTargetName(targetWindow);
+  const request: SerializedMessageRequest = {
+    ...messageRequest,
+    uuid: messageRequest.uuid.getUuidValue(),
+  };
```

Comment:
I am concerned about having the code that knows how to serialize a message request living in the middle of this function in this file. If new properties that need serialization get added, it won't necessarily be obvious to future ENG that they have to go find this code to update it. Would you consider writing a function that lives right next to the message request interfaces (something like `serializeMessageRequest(request: MessageRequestWithRequiredProperties): SerializedMessageRequest`) so that way all the code for dealing with the implementation of the message structures lives together, rather than spread throughout the code?

Reply:
Sorry, I thought I already made a function like this to be extensible in the future. I might have reverted that change by mistake when I got my repo into a bad state. I'll add this in.